src/test/java/com/google/api/codegen/gapic/testdata/csharp/csharp_library.baseline

```diff
@@ -59,6 +64,11 @@ public class DeleteShelfAsyncFlattenedAsyncEmptyResponseTypeWithResponseHandling
 
 // This is a generated sample ("FlattenedAsync", "empty_response_type_without_response_handling")
```

Comment:
Missing the `// DO NOT EDIT!` prefix

Reply:
It's there two lines above this line.

(C# has `DO NOT EDIT` in the license header, so that is removed from this line)

Reply:
Not to keen about changing the format of our boilerplate line, but since that will likely go away real soon, it doesn't matter....